tiledb/CMakeLists.txt

```diff
@@ -220,6 +275,7 @@ target_include_directories(TILEDB_CORE_OBJECTS
     "${TILEDB_CORE_INCLUDE_DIR}"
     "${TILEDB_CORE_INCLUDE_DIR}/tiledb/sm/c_api"
     "${TILEDB_EXTERNALS_INCLUDE_DIRS}"
+    "${CMAKE_CURRENT_BINARY_DIR}/.." # to pickup <build_dir>/tiledb 
```

Comment:
Is this necessary? If so, it seems like something is going wrong in the target-specific include setup.

Reply:
@ihnorton 
With the gen'ing of capn_proto files (tiledb-rest.capnp.<h,c++> into the target build tree, 
that enables finding the tiledb-rest.capnp.h file

Open to any more appropriate way to specify, more details follow...

tiledb/sm/serialization/capnp_utils.h (ref'd by rest_client.cc/array_schema.cc/query.cc/<andonemore>) 
contains
#include "tiledb/sm/serialization/tiledb-rest.capnp.h"

${CMAKE_CURRENT_BINARY_DIR} there is the 'tiledb' at top of target build tree, backing up one level allows finding that header, path being (<buildtree-tiledb-dir>/)tiledb/sm/serialization/tiledb-rest.capnp.h.

Reply:
Ok, I see. Marking resolved for now.